src/EditorFeatures/Core/IntelliSense/AsyncCompletion/CommitManager.cs

```diff
@@ -39,7 +38,7 @@ internal sealed class CommitManager : IAsyncCompletionCommitManager
         private readonly ITextView _textView;
         private readonly IGlobalOptionService _globalOptions;
         private readonly IThreadingContext _threadingContext;
-        private readonly RoslynLSPSnippetExpander _roslynLSPSnippetExpander;
+        private readonly ILanguageServerSnippetExpander? _languageServerSnippetExpander;
```

Comment:
Why is this nullable? won't this always be imported successfully if the VS version is new enough?

Reply:
No, it is exported by a higher layer (EditorFeatures is WPF+Cocoa, but this is only exported for WPF).